src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewViewModel.cs

```diff
@@ -0,0 +1,869 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation.Collections;
+using TreeNodeSelectionState = Windows.UI.Xaml.Controls.TreeViewNode.TreeNodeSelectionState;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal class TreeViewViewModel
+	{
+		private bool m_isContentMode;
+		private SelectedTreeNodeVector m_selectedNodes;
+		private List<object> m_selectedItems;
+		private Dictionary<object, TreeViewNode> m_itemToNodeMap;
+
+		private TreeViewNode m_originNode;
+		private TreeViewList m_TreeViewList;
+
+		public TreeViewViewModel()
+		{
+			//var selectedNodes = new List<TreeViewNode>);
+			//selectedNodes.SetViewModel(*this);
+			//m_selectedNodes.set(* selectedNodes);
+
+			//    var selectedItems = make_self<SelectedItemsVector>();
+			//selectedItems.SetViewModel(*this);
+			//m_selectedItems.set(* selectedItems);
+
+			//    m_itemToNodeMap.set(make<HashMap<IInspectable, TreeViewNode>>());
+		}
+
+		//ViewModel.~ViewModel()
+		//{
+		//	if (m_rootNodeChildrenChangedEventToken.value != 0)
+		//	{
+		//		if (var origin = m_originNode.safe_get())
+		//        {
+		//			get_self<TreeViewNode>(origin).ChildrenChanged(m_rootNodeChildrenChangedEventToken);
+		//		}
+
+		//		ClearEventTokenVectors();
+		//	}
+		//}
+
+		internal void ExpandNode(TreeViewNode value)
+		{
+			value.IsExpanded = true;
+		}
+
+		internal void CollapseNode(TreeViewNode value)
+		{
+			value.IsExpanded = false;
+		}
+
+		//event_token ViewModel.NodeExpanding(const TypedEventHandler<TreeViewNode, IInspectable>& handler)
+		//{
+		//	return m_nodeExpandingEventSource.add(handler);
+		//}
+
+		//void ViewModel.NodeExpanding(const event_token token)
+		//{
+		//	m_nodeExpandingEventSource.remove(token);
+		//}
+
+		//event_token ViewModel.NodeCollapsed(const TypedEventHandler<TreeViewNode, IInspectable>& handler)
+		//{
+		//	return m_nodeCollapsedEventSource.add(handler);
+		//}
+
+		//void ViewModel.NodeCollapsed(const event_token token)
+		//{
+		//	m_nodeCollapsedEventSource.remove(token);
+		//}
+
+		internal void SelectAll()
+		{
+			UpdateSelection(m_originNode, TreeNodeSelectionState.Selected);
+		}
+
+		private void ModifySelectByIndex(int index, TreeNodeSelectionState state)
+		{
+			var targetNode = GetNodeAt(index);
+			UpdateSelection(targetNode, state);
+		}
+
+		internal int Size
+		{
+			get
+			{
+				//	var inner = GetVectorInnerImpl();
+				//	return inner.Size();
+				throw new NotImplementedException();
+			}
+		}
+
+		private object GetAt(uint index)
+		{
+			//TreeViewNode node = GetNodeAt(index);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)